src/types/builtin/route.rs

```diff
@@ -426,43 +426,59 @@ impl RawRouteWithDeltas {
                 .raw_message
                 .0
                 .aspath()
+                .ok()
+                .flatten()
                 .map(|p| p.to_hop_path())
                 .map(TypeValue::from),
             RouteToken::OriginType => {
-                self.raw_message.raw_message.0.origin().map(TypeValue::from)
+                self.raw_message.raw_message.0.origin().ok().flatten().map(TypeValue::from)
             }
             RouteToken::NextHop => self
```

Comment:
This is probably wrong! Getting the next hop for this route from the PDU is not going to work without metadata for this data to which part of the PDU it applies, I assume